packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -87,20 +93,54 @@ const QueueRoomGroup = styled.div`
   align-items: center;
   margin-bottom: 24px;
 `;
+
+const DisableQueueButton = styled(QueueInfoColumnButton)`
+  color: white;
+  background: #da3236;
+  bottom: 0;
+  position: absolute;
+  &:hover,
+  &:focus {
+    color: white;
+    background: #f76c6c;
+  }
+`;
+
 interface QueueInfoColumnProps {
   queueId: number;
+  isTA: boolean;
   buttons: ReactNode;
 }
 
 export function QueueInfoColumn({
   queueId,
+  isTA,
```

Comment:
This prop should probably be `isTaOrProf` or `isStaff` - we want profs to be able to disable as well.